rlberry/agents/ppo/ppo.py

```diff
@@ -299,7 +300,8 @@ def _update(self):
         for _ in range(self.k_epochs):
 
             # shuffle samples
-            rd_indices = np.random.choice(n_samples, size=n_samples, replace=False)
+            np_random = seeding.get_rng()
```

Comment:
`get_rng()` should be in `__init__()`, the agent should have a single `rng` object:

```python
self.rng = seeding.get_rng()   # in __init__()
# ...
self.rng.choice(...)   # where you need it
```